go.mod

```
@@ -9,19 +9,24 @@ require (
 	k8s.io/api v0.31.2
 	k8s.io/apimachinery v0.31.2
 	k8s.io/apiserver v0.31.2
+	k8s.io/client-go v0.31.2
+	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
 	k8s.io/kubernetes v1.31.2
```

As long as we're touching go.mod versions, maybe upgrade to the next patch version of kubernetes APIs?

```suggestion
	k8s.io/api v0.31.3
	k8s.io/apimachinery v0.31.3
	k8s.io/apiserver v0.31.3
	k8s.io/client-go v0.31.3
	k8s.io/kube-openapi v0.0.0-20240228011516-70dd3763d340
	k8s.io/kubernetes v1.31.3
```

I for one am a little surprised mintmaker hasn't auto-upgraded this.  Maybe I can look into adding that functionality.

---

I'd say, let see if we can have mintmaker doing it in another PR